2023-RADIUSS-AWS/JupyterNotebook/aws/eksctl-radiuss-tutorial-2023.yaml

```diff
@@ -33,13 +33,18 @@ iam:
               - "ec2:DetachVolume"
             Resource: '*'
 
-availabilityZones: ["us-east-1a", "us-east-1b", "us-east-1c"]
+availabilityZones:
+  - us-east-1a
+  - us-east-1b
+  - us-east-1c
+
 managedNodeGroups:
   - name: ng-us-east-1a
     instanceType: m6a.8xlarge
-    volumeSize: 300
-    volumeType: io1
-    volumeIOPS: 15000
+    volumeSize: 256
+    volumeType: gp3
+    volumeIOPS: 16000
+    volumeThroughput: 512
     desiredCapacity: 1
     minSize: 1
     maxSize: 6
```

Comment:
Is the maxSize of 6 here sufficient for the number of nodes? E.g., I imagine you calculated this based on the instance type and pods / node, just want to double check!

Reply:
yeah, this will give us support for approximately 250 2 CPU, 4G memory pods if we need to use all 18 nodes. I added potential additional capacity in each AZ in case one or more of them can't get the nodes we need.

Reply:
`m6a.8xlarge` has 32 vCPU and 128GB memory and is a good balance of cheap and powerful with EBS optimizations to guarantee the sustained IOPs the hub needs.